src/queries/users.ts

```diff
@@ -0,0 +1,32 @@
+import { createConnection, getRepository } from "typeorm";
+import { User } from "../entity/User";
+import * as bcrypt from "bcrypt";
+import * as jwt from "jsonwebtoken";
+import {
+  ValidationError,
+  BadCredentials,
+  NotFound,
+} from "../types-and-classes/errors";
+
+export const users = async (_, args, context) => {
+  jwt.verify(context.authScope, process.env.JWT_SECRET);
+  const response = await getRepository(User).findAndCount({
+    select: ["name", "email", "id", "birthDate"],
+    order: {
+      name: "ASC",
+    },
+    skip: args.offset,
+    take: args.limit,
+  });
+  const hasPreviousPage = args.offset > 0 ? true : false;
+  const hasNextPage =
+    args.offset + args.limit < Number(response[1]) ? true : false;
```

Comment:
Geralmente quando vc usa ternário ou if vc fornece uma expressõa booleana para ele comparar. Quando vc quer basicamente um boolean, vc pode usar o próprio resultado da expressão booleana, certo?

```suggestion
  const hasPreviousPage = args.offset > 0;
  const hasNextPage = args.offset + args.limit < response[1];
```

(Acho que não precisa converter o `response1` pra number pq ele já vem como number, não?)